Fixed full screen player behavior & Added “Jump to Holodex” button

P-man2976 opened the pull request:
1. Changed queue context menu from “Add to queue” to “Remove from queue”
2. PlayerBar now changes bar color when hovering
3. Added “Jump to Holodex” button in Video page (https://discord.com/channels/796190073271353385/801760512907935764/961167308225658940)

RiceCakess commented on src/components/player/PlayerBar.tsx:
```diff
@@ -377,6 +376,7 @@ export const PlayerBar = React.memo(
                         {playlistTotalQueue && playlistTotalQueue.length > 0 ? (
                           <SongTable
                             songs={playlistTotalQueue}
+                            menuId={QUEUE_MENU_ID}
```
This isn't supported, because it will only remove from the queue and not the playlist

P-man2976 replied:
I see. I’ll fix it and fix more problem about the player.

P-man2976 commented:
1. "Remove from Queue" button is now displayed only when a song is in the queue.
2. Added an expand button to the right of PlayerBar to expand the player for large displays.
3. Fixed a bug where selecting a song in the playlist would select a song in the queue if the queue and playlist existed in upcoming list at the same time

https://user-images.githubusercontent.com/70518694/162611327-1adbce19-4e31-4cce-9899-425557677869.mov

RiceCakess commented:
Nice work, Thanks!!
